index.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 /* eslint no-var: 0 */
 var once = require('once');
+
 var setups;
 
 exports.setups = function (fns) {
-	setups = fns.map(function (fn) {
+	setups = fns.map(fn => {
```

Comment:
It happens to me: node version v4.2.2  
index.js:3:1
  ✖  3:1   Expected empty line after require statement not followed by another require.  import/newline-after-import
  ✖  7:19  Unexpected function expression.                                               prefer-arrow-callback